### PR TITLE
MAUI-843844-Added center changed event in Maps UG and You tube video updated

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
### **Description**

- Center changed event updated in SfMaps UG.

- You tube getting started video added for Linear and circular progress bar UG

- You tube zoom and pan video added for Maps UG

### CODE REVIEW

File: MAUI/Maps/tile-layer.md [hunk 1]
```diff
@@ -213,4 +213,43 @@ tileLayer.DeleteTilesFromCache();
...
 
-N> You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations. You can also explore our [`.NET MAUI Maps Tile layer example`](https://github.com/syncfusion/maui-demos/) which shows how to configure a Maps in .NET MAUI.
\ No newline at end of file
+N> You can refer to our [.NET MAUI Maps](https://www.syncfusion.com/maui-controls/maui-maps) feature tour page for its groundbreaking feature representations. You can also explore our [`.NET MAUI Maps Tile layer example`](https://github.com/syncfusion/maui-demos/) which shows how to configure a Maps in .NET MAUI.
+
+##  Event
+
+The [`CenterChanged`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapTileLayer.html#Syncfusion_Maui_Maps_MapTileLayer_CenterChanged) event is triggered while zooming and panning the maps.
+
+The following arguments can be obtained from the `MapTileLayer_CenterChanged` event:
```
Comment (M-A32) [inline MAUI/Maps/tile-layer.md:222]: The following arguments can be obtained from the `MapTileLayer_CenterChanged` event.

File: MAUI/CircularProgressBar/getting-started.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 This section explains the steps required to add the circular progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill, and track fill. This section covers only the basic features needed to know and gets started with the Syncfusion circular progress bar.
 
+To get start quickly with our .NET MAUI Circular ProgressBar, you can check the below video.
```
Comment (M-A32) [inline MAUI/CircularProgressBar/getting-started.md:14]: To get start quickly with our .NET MAUI Circular ProgressBar, check the following video.

**Review by M-A32 — CHANGES_REQUESTED** [review]
I have completed the review.

File: MAUI/CircularProgressBar/getting-started.md (hunk 2 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/CircularProgressBar/getting-started.md]: Content modified.

File: MAUI/Maps/tile-layer.md (hunk 1 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Maps/tile-layer.md]: Content modified.

File: MAUI/LinearProgressBar/getting-started.md [hunk 3]
```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 This section explains the steps required to add the linear progress bar control with the progress and its customizable elements such as indeterminate, segment, progress fill, and track fill. This section covers only the basic features needed to know and gets started with the Syncfusion linear progress bar.
 
+To get start quickly with our .NET MAUI Linear ProgressBar, you can check the below video.
```
[the remark this answers is not in the document]
Comment (Priyadharshini-Periyasamy) [reply MAUI/LinearProgressBar/getting-started.md]: Content modified.

**Review by Sasikumar3595 — APPROVED** [review]
Changes are fine

**Pull request merged**